packages/build/src/release.ts

```diff
@@ -25,6 +27,16 @@ export default async function release(
   });
 
   const githubRepo = new GithubRepo(config.repo, octokit);
+  const commitTag = await githubRepo.getTagByCommitSha(config.revision);
+
+  config = {
+    ...config,
+    version: await getReleaseVersionFromTag(commitTag?.name) || config.version
+  };
+
+  // updates the version of internal packages to reflect the tagged one
+  bumpNpmPackages(config.version);
```

Comment:
I think we only want to do these things in the `comand === 'publish'` conditional

Reply:
This is intentional, i think packages consider their package.json version as the current mongosh version, so in the package 'phase' we need the version to be reflected.